codec-http2/src/test/java/io/netty/handler/codec/http2/HpackEncoderTest.java

```diff
@@ -204,6 +204,44 @@ public void testManyHeaderCombinations() throws Http2Exception {
         }
     }
 
+    @Test
+    public void testSanitization() throws Http2Exception {
```

Comment:
My understanding of the small value is that it wont sanitize anything since it will never yield a value greater than 255, making it not a very interesting test. The 300 case should cover what we're really interested in wrt ensuring we don't get wrap-around values of null etc.
A second type of test is that would be valuable is that we should get the same result whether we perform the Huffman encoding or not. What if we just test the size 300 and test it twice: once with the encoded set to 512 (the default I believe) and once super short such to force Huffman encoding, then make sure they both yield the same results?

Reply:
Thanks Bryce for the suggestion. I think it's a good idea and I have updated this PR with this idea implemented. PTAL.